recipes/_filebeat-git.rb

```diff
@@ -0,0 +1,115 @@
+file "#{node['filebeat']['base_dir']}/filebeat.xml" do
+  action :delete
+end
```

Comment:
Ok, the other recipes do this. I don't know why, but i guess the operation is idempotent.